test/testcases/test_web_api/test_document_app/test_paser_documents.py

```
@@ -88,8 +88,8 @@ class TestDocumentsParse:
         [
             pytest.param(None, 101, "required argument are missing: doc_ids, run; ", marks=pytest.mark.skip),
             pytest.param({"doc_ids": [], "run": "1"}, 0, "", marks=pytest.mark.p1),
-            pytest.param({"doc_ids": ["invalid_id"], "run": "1"}, 109, "No authorization.", marks=pytest.mark.p3),
-            pytest.param({"doc_ids": ["\n!?。；！？\"'"], "run": "1"}, 109, "No authorization.", marks=pytest.mark.p3),
+            pytest.param({"doc_ids": ["invalid_id"], "run": "1"}, 109, "no authorization", marks=pytest.mark.p3),
+            pytest.param({"doc_ids": ["\n!?。；！？\"'"], "run": "1"}, 109, "no authorization", marks=pytest.mark.p3),
```

Ruff (0.15.21)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)